apps/web/pages/apps/installed.tsx

```diff
@@ -94,52 +95,48 @@ function ConnectOrDisconnectIntegrationButton(props: {
 }
 
 interface IntegrationsContainerProps {
+  items: inferQueryOutput<"viewer.integrations">["items"];
   variant: App["variant"];
   className?: string;
 }
 
-const IntegrationsContainer = ({ variant, className = "" }: IntegrationsContainerProps): JSX.Element => {
+const IntegrationsContainer = ({
+  items,
+  variant,
+  className = "",
+}: IntegrationsContainerProps): JSX.Element => {
   const { t } = useLocale();
-  const query = trpc.useQuery(["viewer.integrations", { variant, onlyInstalled: true }], { suspense: true });
-
```

Comment:
This should not be a problem, from React-query: [Deduping multiple requests for the same data into a single request](https://react-query.tanstack.com/overview#motivation)

If the query is not being deduped then something else is wrong. Let's focus on that.